packages/browser/src/integrations/offline.ts

```diff
@@ -0,0 +1,116 @@
+import { addGlobalEventProcessor, getCurrentHub } from '@sentry/core';
+import { Event, Integration } from '@sentry/types';
+import localforage from 'localforage';
+
+import { captureEvent } from '../index';
```

Comment:
Use `captureEvent` from `@sentry/minimal`. This way can make this integration work on node.js and on browser if necessary by using something like https://github.com/sindresorhus/is-online